### ISSUE TITLE
qemu fails "dyld: lazy symbol binding failed: Symbol not found: _clock_gettime"

### ISSUE THREAD

**Issue by draeath** [Homebrew/homebrew-core#6460]
Qemu, when compiled, does not work. Qemu installation command: `brew install qemu --with-gtk+ --with-sdl --with-libssh2 --with-vde`

Error output:

```
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /usr/local/bin/qemu-system-x86_64
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _clock_gettime
  Referenced from: /usr/local/bin/qemu-system-x86_64
  Expected in: /usr/lib/libSystem.B.dylib

Trace/BPT trap: 5
```

I suspect this is related to xcode8, as another project failing with similar output (ffmpeg) required a workaround patch (see #4902). Qemu may need similar.

Qemu invocation example:

```
qemu-system-x86_64 \
  -name Windows10 \
  -uuid 51A6F98E-1B57-4A63-B9CF-07B65A4A12F7 \
  -machine pc \
  -cpu qemu64 \
  -smp cpus=1,cores=1,threads=2 \
  -boot order=dc \
  -m size=4G \
  -k en-us \
  -soundhw hda \
  -balloon virtio \
  -drive file=win10-disk0.raw,index=0,media=disk,if=virtio,format=raw \
  -drive file=Windows10.iso,index=2,media=cdrom,if=virtio,readonly \
  -usb \
  -usbdevice tablet \
  -display gtk \                    
  -vga virtio \
  -net nic,model=virtio \
  -net user,vlan=0,hostname=win10vm,dnssearch=draeath.net \
  -rtc base=localtime \
  -S \
  -no-shutdown
```

**Comment by draeath** [comment]
I'm still running El Capitan (10.11.6). Xcode detail:

```
vagante:win10-qemu draeath$ gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 8.0.0 (clang-800.0.38)
Target: x86_64-apple-darwin15.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by Coaxial** [comment]
Same here, while running packer.

installed with `brew install qemu --with-sdl`.

```
==> qemu: Error creating hard drive: QemuImg error: dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
==> qemu:   Referenced from: /usr/local/bin/qemu-img
==> qemu:   Expected in: /usr/lib/libSystem.B.dylib
==> qemu:
==> qemu: dyld: Symbol not found: _clock_gettime
==> qemu:   Referenced from: /usr/local/bin/qemu-img
==> qemu:   Expected in: /usr/lib/libSystem.B.dylib
```

**Comment by ilovezfs** [comment]
@Coaxial mind reporting this upstream to qemu?

**Comment by JCount** [comment]
Should be fixed in #9144.
